src/libraries/sendtohelixhelp.proj

```diff
@@ -80,6 +80,7 @@
 
   <PropertyGroup Condition="'$(TestPackages)' == 'true'">
     <HelixCommand>dotnet msbuild %HELIX_CORRELATION_PAYLOAD%\test.msbuild</HelixCommand>
+    <HelixCommand>$(HelixCommand) /bl:%HELIX_WORKITEM_UPLOAD_ROOT%\msbuild.binlog</HelixCommand>
```

Comment:
I'm curious what folks thoughts are on this.  Should I keep it in? Seems similar to dumps.  Ideally we only upload if there is a failure though.

Reply:
Yes please keep it. We always generate binlogs in CI as remoting onto a machine is often not possible when diagnosing failures in CI.

Reply:
I agree keeping them is good, that said I probably would rename it to something like packageTesting.binlog perhaps?

Reply:
Looking around at things like Xharness and Blazor, lots of work items are invariably uploading binlogs of 40-700 K every day so I don't think this really needs to even consider being conditional unless this is going to be running on on-prem machines where storage pricing changes the equation a bit.    +1 to Joe's comment about a more helpful name.

Reply:
Yep, I changed the name to match the work-item friendly name + package.test.binlog.  That way it behaves better when downloaded. (I think I'm currently at "msbuild (80).binlog" in my downloads folder). 😆